<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Change2</class>
 <widget class="QDialog" name="Change2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1135</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>1311</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 24pt &quot;Arial&quot;;
background-color: qlineargradient(spread:pad, x1:0.499423, y1:0.744, x2:0.5, y2:1, stop:0 rgba(199, 204, 207, 255), stop:0.826923 rgba(245, 251, 255, 255));
color: rgb(175, 75, 135);
</string>
   </property>
   <property name="text">
    <string>      LOGIN APP</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>601</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(44, 44, 44);
font: 700 10pt &quot;Segoe UI&quot;;
background-color: rgb(230, 234, 234);</string>
   </property>
   <property name="title">
    <string>Change password</string>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>390</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 14pt &quot;Segoe UI&quot;;
color: rgb(44, 44, 44);
background-color: qlineargradient(spread:pad, x1:0.37, y1:0.248304, x2:1, y2:1, stop:0 rgba(175, 75, 135, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="text">
     <string>Change password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
color: rgb(175, 75, 135);</string>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_id">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>130</y>
      <width>321</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Segoe UI&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>200</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>9</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Segoe UI&quot;;
color: rgb(44, 44, 44);
background-color: qlineargradient(spread:pad, x1:0.37, y1:0.248304, x2:1, y2:1, stop:0 rgba(175, 75, 135, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="text">
     <string>show</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_repeat">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>310</y>
      <width>321</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial&quot;;
</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>230</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>password should be at least 8 characters include:
Digit , Lowercase ,Uppercase and Special Character</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>86</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 11pt &quot;Arial&quot;;
color: rgb(175,75,135);</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>153</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 11pt &quot;Arial&quot;;
color: rgb(175,75,135);</string>
    </property>
    <property name="text">
     <string>Repeat Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_password">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>321</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial&quot;;
</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>310</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Segoe UI&quot;;
color: rgb(44, 44, 44);
background-color: qlineargradient(spread:pad, x1:0.37, y1:0.248304, x2:1, y2:1, stop:0 rgba(175, 75, 135, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="text">
     <string>show</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>50</y>
      <width>531</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Segoe UI&quot;;
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>lineEdit_password</zorder>
   <zorder>pushButton</zorder>
   <zorder>label_4</zorder>
   <zorder>label</zorder>
   <zorder>lineEdit_id</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>lineEdit_repeat</zorder>
   <zorder>label_8</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>label_12</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
